### DISCUSSION TITLE
Add Walkthrough to Help

### DISCUSSION THREAD

**Discussion by digitalfrost** [processing/p5.js-web-editor#2664] [General]
#### Nature of issue?

- [ ] Found a bug
- [X] Existing feature enhancement
- [ ] New feature request

#### Feature enhancement details:

In the **Help & Feedback** drop-down an FAQ link should be added.

This should link to a documentation page describing common gotchas. For example, how to use external resources.

Such a link will be better for users and will also avoid "polluting" the p5.js documentation with problems such as processing/p5.js#3283

**Comment by digitalfrost** [comment]
This [page](https://blog.codepen.io/documentation/editor/adding-external-resources/) of the CodePen documentation could be a helpful source of inspiration for the FAQ page.

**Comment by catarak** [comment]
I like this idea! I made a video that goes through how to do stuff like this in the web editor, and Dan Shiffman made some too, so at the very least we could link to those.

**Comment by siddhant1** [comment]
@catark should I add them in the FAQ section?
Any particular styling you want?

**Comment by catarak** [comment]
before thinking about styling, i think it makes sense to actually write out the content we want in the FAQ.
